linux/jre/redhat/src/packageTest/java/packaging/RedHatFlavoursWithDnf.java

```diff
@@ -43,25 +43,28 @@ public Stream<? extends Arguments> provideArguments(ExtensionContext context) {
 		 * ubi8 has DNF installed
 		 */
 
-		String containerRegistry = "";
-
-		if (System.getenv("containerRegistry") == null) { 
-			System.out.println("Using docker.io as the default container registry");
-		} else {
-			containerRegistry = System.getenv("containerRegistry");
-			System.out.println("Using container registry: " + containerRegistry);
-		}
-
-		return Stream.of(
-			Arguments.of(containerRegistry + "rockylinux", "8"),
-			Arguments.of(containerRegistry + "fedora", "35"),
-			Arguments.of(containerRegistry + "fedora", "36"),
-			Arguments.of(containerRegistry + "fedora", "37"),
-			Arguments.of(containerRegistry + "fedora", "38"),
-			Arguments.of(containerRegistry + "fedora", "39"),
-			Arguments.of(containerRegistry + "redhat/ubi8", "latest"),
-			Arguments.of(containerRegistry + "redhat/ubi9", "latest"),
-			Arguments.of(containerRegistry + "oraclelinux", "8")
-		);
+		 String quayIoDefaultRegistry = "quay.io/";
+		 String dockerDefaultRegistry = "";
+ 
+		 if (System.getenv("containerRegistry") == null) {
+			 System.out.println("Using docker.io and quay.io as the default container registry");
+		 } else {
+			 quayIoDefaultRegistry = dockerDefaultRegistry = System.getenv("containerRegistry");
+			 System.out.println("Using container registry: " + dockerDefaultRegistry);
+		 }
+ 
+		 return Stream.of(
```

Comment:
I see that centos7 was missing already there.. so the first occurecne was left over?

Reply:
Yeah, I copy/pasted the RedHat flavours w/ DNF so they are all the same so I think there were a few variations. These are just tests not what is actually published which is controlled by the Jenkinsfile